ExampleMod/Content/Buffs/ExampleDefenseBuff.cs

```diff
@@ -8,12 +11,22 @@ public class ExampleDefenseBuff : ModBuff
 		public override void SetDefaults() {
 			DisplayName.SetDefault("Defensive Buff");
 			Description.SetDefault("Grants +4 defense.");
-			Main.buffNoTimeDisplay[Type] = false;
-			Main.debuff[Type] = false; //Add this so the nurse doesn't remove the buff when healing
+			Main.buffNoTimeDisplay[Type] = false; //Set this to true so the remaining buff time should not be displayed (by default it also won't get displayed if buffTime is less than 3).
+			Main.debuff[Type] = false; //Set this to true so the nurse doesn't remove the buff when healing.
 		}
 
 		public override void Update(Player player, ref int buffIndex) {
 			player.statDefense += 4; //Grant a +4 defense boost to the player while the buff is active.
 		}
+
+		//Optional
+		public override bool PreDraw(SpriteBatch spriteBatch, int buffIndex, ref BuffDrawParams drawParams) {
+			//You can use this hook to make something special happen when the buff icon is drawn (such as reposition it, pick a different texture, etc.).
+			//Here we make the icon have a lime green tint.
+			drawParams.drawColor = Color.LimeGreen * Main.buffAlpha[buffIndex];
+			
+			//Return true to let the game draw the buff icon.
+			return true;
+		}
```

Comment:
Adding changes like this, which are unrelated to the file's name, may very well be the reason why the old EM turned in a mess quite quick. I think it'll be fine if you just stack this effect with the one in AnimatedBuff.

Reply:
Good point, I'll move it